components/basic-filters/src/main/java/org/datacleaner/beans/filter/NullCheckFilter.java

```diff
@@ -132,12 +132,12 @@ public Query optimizeQuery(Query q, NullCheckCategory category) {
                 }
                 q.where(column, OperatorType.DIFFERENT_FROM, null);
                 if (considerEmptyStringAsNull && col.getDataType() == String.class) {
-                    q.where(column, OperatorType.DIFFERENT_FROM, "");
+                    q.where(column, OperatorType.LIKE, "%");
```

Comment:
When I try CLOB then variant with both, "NOT_NULL" and "NULL", works fine and variant only with "NOT_NULL" gives me 
```
--- DataCleaner progress information user-log ---
11:30:01 INFO: Job begin
11:30:01 INFO: Starting component 'MESSAGE is null?'
11:30:01 INFO: Starting component 'Value distribution (ID) (MESSAGE is null?=NOT_NULL)'
11:30:01 INFO: Starting component 'Value distribution (MESSAGE) (MESSAGE is null?=NOT_NULL)'
11:30:01ERROR: An error occurred in the analysis job!

Could not execute query: ORA-00932: inconsistent datatypes: expected - got CLOB


org.apache.metamodel.MetaModelException: Could not execute query: ORA-00932: inconsistent datatypes: expected - got CLOB

	at org.apache.metamodel.jdbc.JdbcUtils.wrapException(JdbcUtils.java:70)
	at org.apache.metamodel.jdbc.JdbcDataContext.executeQuery(JdbcDataContext.java:511)
	at org.datacleaner.job.runner.SourceTableRowProcessingPublisher.processRowsInternal(SourceTableRowProcessingPublisher.java:225)
	at org.datacleaner.job.runner.AbstractRowProcessingPublisher.processRows(AbstractRowProcessingPublisher.java:208)
	at org.datacleaner.job.tasks.RunRowProcessingPublisherTask.execute(RunRowProcessingPublisherTask.java:43)
	at org.datacleaner.job.concurrent.TaskRunnable.run(TaskRunnable.java:61)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```